build/crowdin-credentials.js

```diff
@@ -0,0 +1,22 @@
+import path from 'path'
+import fs from 'fs'
+import chalk from 'chalk'
+
+const credentialPlaceholder = 'API_TOKEN_PLACEHOLDER'
+
+const CREDENTIAL = process.env.CROWDIN_TOKEN
+
+; (
```

Comment:
format ?

Reply:
```ts
;(
```

Reply:
This is already a formatted result I suppose. I think I should avoid using this though.